roxie/udplib/udpsha.hpp

```diff
@@ -29,6 +29,11 @@ typedef bool (*PKT_CMP_FUN) (void *pkData, void *key);
 
 #define UDP_SEQUENCE_COMPLETE 0x80000000
 
+// Flag bits in pktSeq field
+#define UDP_PACKET_COMPLETE           0x80000000
+#define UDP_PACKET_RESERVED           0x40000000  // Not used - could move UDP_SEQUENCE_COMPLETE here?
+#define UDP_PACKET_SEQUENCE_MASK     ~0x3fffffff
```

Comment:
@richardkchapman I don't think the ~ is correct

Reply:
No. Kept changing my mind where the ~ belonged!